src/components/Card/Card.story.jsx

```diff
@@ -51,6 +51,27 @@ storiesOf('Watson IoT|Card', module)
       </div>
     );
   })
+  .add('with explicit tooltip and long title', () => {
```

Comment:
Does this actually need a story? I think this could be achieved by making the tooltip a text knob in the other stories that use it

Reply:
In my opinion the explicit tooltip should appear in at least one story and I don't think it can be a knob since it isn't a string but a ReactElement. I've now merged this story with the story  _with ellipsed title and tooltip_ so that it contains both tooltip scenarios.